src/Datasource/EntityTrait.php

```diff
@@ -1066,17 +1066,11 @@ public function getInvalidField($field)
      * or to be able to log it away.
      *
      * @param array $fields The values to set.
-     * @param mixed|null $overwriteValue The invalid value to be set for $fields.
-     * @param bool $overwrite Whether or not to overwrite pre-existing values for $field.
      * @return $this
      */
-    public function setInvalid(array $fields, $overwriteValue = null, $overwrite = false)
+    public function setInvalid(array $fields)
     {
         foreach ($fields as $field => $value) {
-            if ($overwrite === true) {
-                $this->_invalid[$field] = $overwriteValue;
-                continue;
-            }
             $this->_invalid += [$field => $value];
```

Comment:
Looks like you removed the overwrite feature entirely.

I think there should be overwrite support here disabled by default as before.